hack/deployer/runner/tanzu.go

```diff
@@ -187,11 +187,7 @@ func (t *TanzuDriver) ensureWorkDir() error {
 	if workDir == "" {
 		// base work dir in HOME dir otherwise mounting to container won't work without further settings adjustment
 		// in macOS in local mode. In CI mode we need the workdir to be in the volume shared between containers.
-		var err error
-		workDir, err = os.MkdirTemp(os.Getenv("HOME"), t.plan.ClusterName)
-		if err != nil {
-			return err
-		}
+		workDir := filepath.Join(os.Getenv("HOME"), t.plan.ClusterName)
```

Comment:
Absolutely.

Reply:
It is created just after for the case where it was configured in the plan. I wrote ':=' instead of '=' @#!
https://github.com/elastic/cloud-on-k8s/blob/8c36a0ae70fafd7b3c252055496821bd52461281/hack/deployer/runner/tanzu.go#L198